### ISSUE TITLE
Ambiguous class resolution

### ISSUE THREAD

**Issue by seb-jean** [symfony/ux#577]
When I use command :

`symfony composer require symfony/ux-turbo-mercure`

I have errors:

```console
Warning: Ambiguous class resolution, "Symfony\UX\Turbo\Bridge\Mercure\Broadcaster" was found in both "C:/wamp64/www/my_project/vendor/symfony/ux-turbo-mercure/src\Broadcaster.php" and "C:/wamp64/www/my_project/vendor/symfony/ux-turbo/src/Bridge/Mercure/src\Broadcaster.php", the first will be used.
Warning: Ambiguous class resolution, "Symfony\UX\Turbo\Bridge\Mercure\TurboStreamListenRenderer" was found in both "C:/wamp64/www/my_project/vendor/symfony/ux-turbo-mercure/src\TurboStreamListenRenderer.php" and "C:/wamp64/www/my_project/vendor/symfony/ux-turbo/src/Bridge/Mercure/src\TurboStreamListenRenderer.php", the first will be used.
```

Ouput general:
```console
symfony composer require symfony/ux-turbo-mercure
Using version ^2.6 for symfony/ux-turbo-mercure
./composer.json has been updated
Running composer update symfony/ux-turbo-mercure
Loading composer repositories with package information
Updating dependencies
Lock file operations: 8 installs, 0 updates, 0 removals
  - Locking lcobucci/clock (2.2.0)
  - Locking lcobucci/jwt (4.2.1)
  - Locking psr/clock (1.0.0)
  - Locking stella-maris/clock (0.1.7)
  - Locking symfony/mercure (v0.6.2)
  - Locking symfony/mercure-bundle (v0.3.5)
  - Locking symfony/ux-turbo (v2.6.0)
  - Locking symfony/ux-turbo-mercure (v2.6.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 8 installs, 0 updates, 0 removals
  - Downloading psr/clock (1.0.0)
  - Downloading stella-maris/clock (0.1.7)
  - Downloading lcobucci/clock (2.2.0)
  - Downloading symfony/mercure (v0.6.2)
  - Installing psr/clock (1.0.0): Extracting archive
  - Installing stella-maris/clock (0.1.7): Extracting archive
  - Installing lcobucci/clock (2.2.0): Extracting archive
  - Installing lcobucci/jwt (4.2.1): Extracting archive
  - Installing symfony/mercure (v0.6.2): Extracting archive
  - Installing symfony/ux-turbo (v2.6.0): Extracting archive
  - Installing symfony/mercure-bundle (v0.3.5): Extracting archive
  - Installing symfony/ux-turbo-mercure (v2.6.0): Extracting archive
Generating optimized autoload files
Warning: Ambiguous class resolution, "Symfony\UX\Turbo\Bridge\Mercure\Broadcaster" was found in both "C:/wamp64/www/my_project/vendor/symfony/ux-turbo-mercure/src\Broadcaster.php" and "C:/wamp64/www/my_project/vendor/symfony/ux-turbo/src/Bridge/Mercure/src\Broadcaster.php", the first will be used.
Warning: Ambiguous class resolution, "Symfony\UX\Turbo\Bridge\Mercure\TurboStreamListenRenderer" was found in both "C:/wamp64/www/my_project/vendor/symfony/ux-turbo-mercure/src\TurboStreamListenRenderer.php" and "C:/wamp64/www/my_project/vendor/symfony/ux-turbo/src/Bridge/Mercure/src\TurboStreamListenRenderer.php", the first will be used.
135 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Symfony operations: 2 recipes (6d9a582a15c7b7f664ff74ac25fd93d4)
  - Configuring symfony/ux-turbo (>=v2.6.0): From auto-generated recipe
  - Configuring symfony/mercure-bundle (>=0.3): From github.com/symfony/recipes:main
Synchronizing package.json with PHP packages
Don't forget to run npm install --force or yarn install --force to refresh your JavaScript dependencies!

Executing script cache:clear [OK]
Executing script assets:install public [OK]

 What's next?


Some files have been created and/or updated to configure your new packages.
Please review, edit and commit them: these files are yours.
```

**Comment by weaverryan** [comment]
I'm checking into this :)

**Comment by seb-jean** [comment]
Close in favor #578

**Comment by weaverryan** [comment]
For quick reference to others. Once #580 is merged and tagged (as 2.6.1), you will be able to `composer remove symfony/ux-turbo-mercure`, upgrade `symfony/ux-turbo`, then enable the controller in `controllers.json`. See the CHANGELOG in #580 for the full details.
